include/openthread/platform/radio.h

```diff
@@ -183,10 +183,10 @@ struct otExtAddress
 } OT_TOOL_PACKED_END;
 
 /**
- * Represents the IEEE 802.15.4 Extended Address.
+ * Represents the IEEE 802.15.4 Extended Address and EUI64.
  *
  */
-typedef struct otExtAddress otExtAddress;
+typedef struct otExtAddress otExtAddress, otEui64;
```

Comment:
Suggest a different `typedef` for `otEui64` (so doxygen for it is created properly?).

BTW, if we want to add a new `otEui64` type may be good to that we then use in more places when EUI-64 is used.
-For example, in `otLinkGetFactoryAssignedIeeeEui64()` or `otCommissionerAddJoiner()`, ...? 
- These all use `otExtAddress` today. 
- Suggest maybe doing this in its own PR since it can be a bigger change? And in this PR for EUI64 TLV we can still use `otExtAddress`?

Reply:
will reuse otExtAddress in this PR